plans/rpminspect/README.md

```diff
@@ -0,0 +1,48 @@
+# Rpminspect
+
+<!-- SPHINX-START -->
+
+Run [rpminspect] on the current Copr project or Koji build
+
+## Synopsis
+
+```yaml
+plans:
+  import:
+    url: https://github.com/packit/tmt-plans
+    ref: main
+    name: /plans/rpminspect
+```
+
+## Options
+
+`RPMINSPECT_KOJI_BUILD`
+: Run `rpminspect` on the specified Koji build instead of the expected Copr project
```

Comment:
I found [this](https://www.markdownguide.org/extended-syntax/#definition-lists), though it seems like an optional feature… I think you should be able to embed the HTML definition list itself though :thinking:

Reply:
Thanks let me give it a try in their preview. If you can test it on docusaurus as well that would be splendid.

How about [colon_fence](https://myst-parser.readthedocs.io/en/latest/syntax/admonitions.html#other-containers-grids-tabs-cards-etc) and [admonitions](https://myst-parser.readthedocs.io/en/latest/syntax/admonitions.html#admonition-types)? Those are meant to play well with github-flavor markdown, i.e. the inner part is still parsed, while the decorations have minimal cosmetic distraction.

Reply:
Admonitions are [supported](https://docusaurus.io/docs/markdown-features/admonitions), don't forget the new lines because of the pre-commit formatting, and can put it in one line and break the parsing :D

Reply:
Great, does docusaurus support both syntax `:::note` and `:::{note}` or only the former? `myst` is using the latter.

I have just tested the `<dt>`, and at least with sphinx there is a problem that the inner part doesn't get rendered by markdown anymore :(

Reply:
I have raised a [feature-request](https://docusaurus.io/feature-requests/p/deflist-definition-list) about the `deflist` upstream, if you can vote on it to get it some attention. Seems like something easy to implement diven the syntax, but I didn't look into their code-base.

In the meantime, would it be ok to implement as if we had `deflist` support?

Reply:
Prettier disagrees :((
[Upstream issue](https://github.com/prettier/prettier/issues/10701). Will just leave it there for now and fix it when the support arrives